create_aio_app/main.py

```diff
@@ -11,16 +12,23 @@ def main():
     args = parse_arguments()
     template_path = str(parent / 'template')
 
-    if not args.get('name'):
-        result = cookiecutter(template_path)
-    else:
-        ctx = {
-            'project_name': args.get('name'),
-            'use_postgres': 'n' if args.get('without_postgres') else 'y',
-            'use_redis': 'y' if args.get('redis') else 'n',
+    kwargs = {}
+
+    if args.get('name'):
+        kwargs = {
+            'no_input': True,
+            'extra_context': {
+                'project_name': args.get('name'),
+                'use_postgres': 'n' if args.get('without_postgres') else 'y',
+                'use_redis': 'y' if args.get('redis') else 'n',
+            },
         }
 
-        result = cookiecutter(template_path, no_input=True, extra_context=ctx)
+    try:
+        result = cookiecutter(template_path, **kwargs)
+    except OutputDirExistsException as exc:
+        print('\n\n\033[91mDirectory with such name already exists!\033[00m\n')
```

Comment:
I'd strongly advise using constants or better some library for colors rather then hardcoding unreadable sequences. It's not perl!

Reply:
I agree, but it is just few colorized lines:) I can add colorama or something if it seems reasonable but as for me sit is unnecessary at this point, but moving to const sounds good. 👌🏻

Reply:
cookiecutter has `click` in requirements, so u can rewrite to something like that

```python
import click

click.echo(click.style('Directory with such name already exists!', fg='green'))
```

Reply:
As for me relying on implicit dependency is not very good plan. Maybe @webknjaz has opinion about such way?

Reply:
So, add this library to `install_requires`.

Reply:
I meant not actually this, but as you wish.

Reply:
> but it is just

@hzlmn So it's okay if I accidentally remove, let's say, 0 from there? Would it be obvious what's the breaking change?